### PR TITLE
Port pure pursuit

### PR DESCRIPTION
**Pull request by rej55**
## PR Type

- Improvement

## Description

Porting pure_pursuit module.
See https://github.com/autowarefoundation/autoware.universe/pull/271

## Review Procedure

Change following arguments from mpc_follower to pure_pursuit and run planning simulator.
https://github.com/tier4/AutowareArchitectureProposal_launcher/blob/3c6700ceb078f5d08feacec1bd876bcd1f7f9815/autoware_launch/launch/planning_simulator.launch.xml#L52

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: control_launch/launch/control.launch.py [hunk 1]
```diff
@@ -53,24 +65,41 @@ def launch_setup(context, *args, **kwargs):
...
+            remappings=[
+                ("input/reference_trajectory", "/planning/scenario_planning/trajectory"),
+                ("input/current_odometry", "/localization/kinematic_state"),
+                ("output/control_raw", "lateral/control_cmd"),
+            ],
+            parameters=[
+                lat_controller_param,
+            ],
+            extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+        )
```
Comment (tkimura4) [inline control_launch/launch/control.launch.py]: I think using IfCondition (or LaunchCOnfigurationEquals?) is better.
https://github.com/tier4/AutowareArchitectureProposal_launcher/blob/5527a3b1541466aad4586128cb4d1fa715143a12/planning_launch/launch/scenario_planning/lane_driving/motion_planning/motion_planning.launch.py#L182-L185

Comment (rej55) [reply control_launch/launch/control.launch.py]: Fixed.
https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/166/commits/9eb3b64329f38fa4386cedc4df9e19a80a031a5c

**Comment by tkimura4** [comment]
RvizのDebug markerのトピック変更お願いします

/control/trajectory_follower/pure_pursuit/debug/markers
->
/control/trajectory_follower/pure_pursuit_node_exe/debug/markers

**Comment by rej55** [comment]
@tkimura4 
修正しました https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/166/commits/5a211a96470a9f91de0855b3a2f96099b0447ba4

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**